genie-docs/src/docs/asciidoc/concepts/_security.adoc

```diff
@@ -20,7 +20,7 @@ then be accessed via Genie code if necessary.
 
 All the security can be enabled or disabled as desired by system administrators. You can find the various properties
 under `genie.security.*` in the <<Properties>> section. Spring specific properties are also involved but the enabling
-and disabling is done via the `genie.security` properties.
+and disabling is done via the `security.*` properties.
```

Comment:
You need to back this one out... enabling and disabling the current security profiles are done via the genie.security path.